src/adapter/tests/parameters.rs

```diff
@@ -93,6 +93,15 @@ fn test_github_18189() {
     assert_eq!(sync.get("allowed_cluster_replica_sizes"), "");
 }
 
+#[test]
+fn test_vars_are_synced() {
+    let vars = SystemVars::default();
+    let sync = SynchronizedParameters::new(vars);
+
+    // A smoke test to ensure the variables we want to get synced, are getting synced
+    assert!(sync.is_synchronized("enable_force_introspection_cluster"));
+}
```

Comment:
This is probably not the right file to include this. It's for query parameters (like the test below). Though it seems that `test_github_18189` also found it's way to this file, even though it doesn't belong.

Would you mind finding a new home for both?

Reply:
Done! Made these unit tests in the `adapter::config::params` module